CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Features
 
+- `[@jest/expect]` provide type of `actual` as a generic argument to `Matchers` to allow better-typed extensions (#TBD)
```

Comment:
I think this can go under fixes as well.

```suggestion
- `[expect, @jest/expect]` Provide type of `actual` as a generic argument to `Matchers` to allow better-typed extensions ([#13848](https://github.com/facebook/jest/pull/13848))
```